.github/workflows/unstable-release.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish unstable release
+on:
+  workflow_dispatch: 
+    inputs:
+      tag:
+        description: 'Tag name'
+        required: true
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: start X server
+      run: |
+        Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+        echo "DISPLAY=:99" >> $GITHUB_ENV
+    - uses: actions/checkout@v2
+    - name: read version
+      id: version
+      uses: martinbeentjes/npm-get-version-action@v1.3.1
+    - name: ci
+      run: npm ci
+    - name: test
+      run: npm test
+    - name: package
+      run: npm run package
+    - name: rename
+      run: mv vscode-clangd-${{ steps.version.outputs.current-version }}.vsix
+              vscode-clangd-unstable-${{ steps.version.outputs.current-version }}.vsix
```

Comment:
If we're going to make a frequent unstable release (like per commit), I think it would be clearer to use the commit in the name, rather than use the current release version in the package.json (not sure this is possible to get the commit in workflow.)

Reply:
Makes sense. I updated the action to do this.